portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,40 +36,56 @@
 public class DataServlet extends HttpServlet {
     
 private static final ArrayList<String> messagesList = new ArrayList<String>();
+private static final String index = "/index.html";
+private static final String comment = "Comment";
+private static final String content = "content";
+private static final String textInput = "text-input";
+
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<Task> tasks = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String content = (String) entity.getProperty("content");
+
+      Task task = new Task(id, content);
+      tasks.add(task);
+    }
+
     Gson gson = new Gson();
-    String json = gson.toJson(messagesList);
+    String json = gson.toJson(tasks);
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
+
 @Override
```

Comment:
nit: Indent +2

Reply:
I'm not sure I understand what this is referring to. Could you clarify?

Reply:
Sure - add 2 spaces before `@Override` so that it is aligned with `public void ...` below.